data/Ritisha542.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "Ritisha Sen",
+  "type": "personal",
+  "bio": "CS Student",
+  "links": [
+    {
+      "name":"Follow me on Github",
+      "url": "https://github.com/Ritisha542",
+      "icon":"FaGithub"
+    },
```

Comment:
Hi @Ritisha542,
Thank you for contributing to Linkfree.

It seems that the JSON is invalid.
You need to close the links array.

You can test you JSON code in playgroud before making a PR.
-> https://linkfree.io/playground

Thanks :)